test/e2e/provisioning/testdata/machinedeployment-nutanix.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineDeployment
+metadata:
+  name: << MACHINE_NAME >>
+  namespace: kube-system
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      name: << MACHINE_NAME >>
+  template:
+    metadata:
+      labels:
+        name: << MACHINE_NAME >>
+    spec:
+      providerSpec:
+        value:
+          sshPublicKeys:
+            - "<< YOUR_PUBLIC_KEY >>"
+          cloudProvider: "nutanix"
+          cloudProviderSpec:
+            username: '<< NUTANIX_USERNAME >>'
+            password: '<< NUTANIX_PASSWORD >>'
+            endpoint: '<< NUTANIX_ENDPOINT >>'
+            proxyURL: '<< NUTANIX_PROXY_URL >>'
+            allowInsecure: true
+            clusterName: '<< NUTANIX_CLUSTER >>'
+            projectName: '<< NUTANIX_PROJECT >>'
+            subnetName: '<< NUTANIX_SUBNET >>'
+            imageName: 'machine-controller-e2e-<< OS_NAME >>'
+            cpus: 2
+            memoryMB: 2048
+            diskSize: 20
+          operatingSystem: "<< OS_NAME >>"
+          operatingSystemSpec:
+            distUpgradeOnBoot: false
+            disableAutoUpdate: true
+            attachSubscription: false
```

Comment:
Why do you need this here? This is for RHEL thus I would remove it

Reply:
Copied from another `MachineDeployment` testdata YAML. I'll remove it.